CHANGELOG.md

```diff
@@ -150,6 +150,9 @@ customers cannot upgrade their bootloader, its changes are recorded separately.
 
 ## Bootloader
 
+### [Unreleased]
+- Implement OP_HARDWARE endpoint, to identify the SC model
```

Comment:
SC=secure chip. Imho abbreviations just make it harder to read, not everyone knows what it means.